Providers/Makefile

```diff
@@ -100,6 +101,7 @@ PROVIDERS+=nxMySqlUser
 PROVIDERS+=nxAvailableUpdates
 PROVIDERS+=nxFileInventory
 PROVIDERS+=nxOMSAutomationWorker
+PROVIDERS+=nxOMSContainers
```

Comment:
Check with @D1v38om83r but I don't think you will use this provider in the non-OMS case.